internal/commands/user/inputs.go

```diff
@@ -0,0 +1,115 @@
+package user
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	userDataEmail = "email"
+	userDataName  = "name"
+)
+
+// usersInputs are the filtering inputs for a user command
+type usersInputs struct {
+	State         userStateType
+	ProviderTypes []string
+	Status        statusType
+	Users         []string
+}
+
+// ResolveUsers will use the provided Realm client to resolve the users specified by the realm.App through inputs
+func (i *usersInputs) ResolveUsers(ui terminal.UI, client realm.Client, app realm.App) ([]realm.User, error) {
+	var err error
+
+	if len(i.Users) > 0 {
+		var users []realm.User
+		for _, userID := range i.Users {
+			foundUser, err := client.FindUsers(app.GroupID, app.ID, realm.UserFilter{IDs: []string{userID}})
```

Comment:
[question] why aren't we just simply creating a `realm.UserFilter` from the `usersInputs` here and sending that in to `realmClient.FindUsers`?  a lot of this code seems to be duplicating logic we already have defined in the realm client?